### PR TITLE
Cleans up some timer messages

### PR DESCRIPTION
**Pull request by ShizCalev**

### CODE REVIEW

File: code/__HELPERS/time.dm [hunk 1]
```diff
@@ -64,7 +64,7 @@ GLOBAL_VAR_INIT(rollovercheck_last_timeofday, 0)
 //Takes a value of time in deciseconds.
 //Returns a text value of that number in hours, minutes, or seconds.
 /proc/DisplayTimeText(time_value, truncate = FALSE)
-	var/second = time_value*0.1
+	var/second = (time_value)*0.1
```
Comment (SpaceManiac) [inline code/__HELPERS/time.dm]: any significance?

File: code/__HELPERS/time.dm [hunk 2]
```diff
@@ -144,7 +144,7 @@ GLOBAL_VAR_INIT(rollovercheck_last_timeofday, 0)
 			else if(day && (!minute || !second))
 				hour = " and 1 hour"
 			else
-				hour = "[truncate ? "hour" : "1 hour"]"
+				hour = "[truncate ? "hour" : "1 hour"]"
```
Comment (SpaceManiac) [inline code/__HELPERS/time.dm]: line endings, or?

Comment (ShizCalev) [reply code/__HELPERS/time.dm]: yeah

File: code/__HELPERS/time.dm (hunk 1 above)
Comment (ShizCalev) [reply code/__HELPERS/time.dm]: Shouldn't have any. Just added it for piece of mind.

**Review by SpaceManiac — APPROVED** [review]

**Pull request merged**